README.md

```diff
@@ -1335,6 +1338,306 @@ You can either provide a date range or a single date. If you provide a single da
   ]
 }
 ```
+</details>
+
+### Schoolgegevens: `GET /rest/v1/[id]/schoolgegevens`
+<details><summary>Click to open</summary>
+
+Fetches info about the school, including your mentor.
+
+#### Parameters
+
+| Name                       | Type      | Value                 |
+|----------------------------|-----------|-----------------------|
+| Authorization              | Header    | Bearer [access_token] |
+
+#### Returns
+
+```json
+{
+    "$type": "leerling.RLeerlingSchoolgegevens",
+    "instellingsnaam": "REDACTED",
+    "vestigingsnaam": "REDACTED",
+    "plaats": "REDACTED",
+    "straat": "REDACTED",
+    "postcode": "REDACTED",
+    "telefoonnummer": "REDACTED",
+    "email": "REDACTED",
+    "leerjaar": 99,
+    "mentoren": [
+        "REDACTED"
+    ]
+}
+```
+
+</details>
+
+### Vakanties: `GET /rest/v1/vakanties/leerling/[id]`
+<details><summary>Click to open</summary>
+
+Fetches info about the school, including your mentor.
+
+#### Parameters
+
+| Name                       | Type      | Value                 |
+|----------------------------|-----------|-----------------------|
+| Authorization              | Header    | Bearer [access_token] |
+
+#### Returns
+
+```json
+{
+    "items": [
+        {
+            "$type": "participatie.RVakantie",
+            "links": [
+                {
+                    "id": 123456789,
+                    "rel": "self",
+                    "type": "participatie.RVakantie",
+                    "href": "https://api.somtoday.nl/rest/v1/vakanties/123456789"
+                }
+            ],
+            "permissions": [
+                {
+                    "full": "participatie.RVakantie:READ:INSTANCE(123456789)",
+                    "type": "participatie.RVakantie",
+                    "operations": [
+                        "READ"
+                    ],
+                    "instances": [
+                        "INSTANCE(123456789)"
+                    ]
+                }
+            ],
+            "additionalObjects": {},
+            "naam": "Herfstvakantie",
+            "beginDatum": "2023-10-16T00:00:00.000+02:00",
+            "eindDatum": "2023-10-20T00:00:00.000+02:00"
+        },
+        ...
+    ]
+}
+```
+
+</details>
+
+### Studiemateriaal: `GET /rest/v1/vakanties/leerling/[id]` & `GET /rest/v1/vakanties/leerling/[id]/vak[uuid]`
```

Comment:
I do not think that the urls in this portion of the docs are correct. 

If i'm correct the urls should be:

`/rest/v1/vakken/studiemateriaal/[leerling id]` for all subjects with studiemateriaal
`/rest/v1/studiemateriaal/algemeen/[leerling id]` for the general studiemateriaal (I think that the links to online books are here, but I'm not sure.
`/rest/v1/studiemateriaal/[leerling id]/vak/[vak id]` for the jaarbijlagen, lesstof etc.

so go and check that out :)